featuretools/primitives/__init__.py

```diff
@@ -1,2 +1,8 @@
 # flake8: noqa
 from .api import *
+
+import pkg_resources
+for entry_point in pkg_resources.iter_entry_points('featuretools_primitives'):
+    if hasattr(entry_point, 'primitives'):
```

Comment:
should this be `hasattr(entry_point.load(), 'primitives')`?

Reply:
Fixed.